src/internal/operators/skipLast.ts

```diff
@@ -5,76 +5,92 @@ import { operate } from '../util/lift';
 import { OperatorSubscriber } from './OperatorSubscriber';
 
 /**
- * Skip the last `count` values emitted by the source Observable.
+ * Skip a specified number of values before the completion of an observable.
  *
  * ![](skipLast.png)
  *
- * `skipLast` returns an Observable that accumulates a queue with a length
- * enough to store the first `count` values. As more values are received,
- * values are taken from the front of the queue and produced on the result
- * sequence. This causes values to be delayed.
+ * Returns an observable that will emit values as soon as it can, given a number of
+ * skipped values. For example, If you `skipLast(3)` on a source, when the source
+ * emits its fourth value, the first value the source emitted will finally be emitted
+ * from the returned observable, as it is no longer part of what needs to be skipped.
+ *
+ * All values emitted by the result of `skipLast(N)` will be delayed by `N` emissions,
+ * as each value is held in a buffer until enough values have been emitted that that
+ * the buffered value may finally be sent to the consumer.
+ *
+ * After subscribing, unsubscribing will not result in the emission of the buffered
+ * skipped values.
  *
  * ## Example
- * Skip the last 2 values of an Observable with many values
+ *
+ * Skip the last 2 values of an observable with many values
```

Comment:
Maybe add `:` or `.` at the end of sentence.

Reply:
Now I see that many examples in many other files don't have none of what I proposed here.